### PR TITLE
overrides for gdal, moto, pdal, pyproj, others

### PR DESCRIPTION
**Pull request by rskew**

### CODE REVIEW

File: overrides.nix [hunk 1]
```diff
@@ -920,6 +951,22 @@ self: super:
...
   );
 
+  pyproj = super.pyproj.overridePythonAttrs (
+    old: {
+      buildInputs = (old.buildInputs or [ ]) ++
+        [ self.cython ];
+      PROJ_DIR = "${pkgs.proj}";
+      PROJ_LIBDIR = "${pkgs.proj}/lib";
+      # pkgs.proj doesn't export the required headers, so grab them from the source repo
+      PROJ_INCDIR = "${pkgs.fetchFromGitHub {
```
Comment (adisbladis) [inline overrides.nix:961]: This needs to be changed to something version independent. We don't want to maintain fixed upstreams in our overrides.

Preferably you'd fix this in nixpkgs to include the relevant headers and only then use it in poetry2nix.
Feel free to CC me on any nixpkgs PR.

**Review by adisbladis — CHANGES_REQUESTED** [review]

File: overrides.nix (hunk 1 above)
Comment (rskew) [reply overrides.nix]: Turns out `pks.proj.dev` has the headers in it already :tada:

**Comment by rskew** [comment]
Oops, I think I force pushed over your force push!

How are you running `nixpkgs-fmt`? When I run it it creates a huge diff (beyond my changes)

**Comment by adisbladis** [comment]
I think maybe you're on an older version of `nixpkgs-fmt`? It's not been super stable in it's formatting output.

**Comment by rskew** [comment]
That could be it.
Cheers :)

**Pull request merged**